modules/ROOT/pages/osv2-usage.adoc

```diff
@@ -1,27 +1,83 @@
-= View Usage Graphs
+= Rate Limiting and Billing
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 :keywords: osv2, os2, object store, store, usage
 
+Object Store v2 uses two metrics to determine billing and rate-limiting:
```

Comment:
If you wrote something like this in this short description, you could clarify the relationships and get rid of the links I've flagged.

"...rate limiting: transactions per second for rate limiting, and effective API requests for billing calculation."

It's slightly confusing that the lead-in sentence leads to  the unmentioned API items immediately below it. 

rate limiting isn't a compound word yet AFAIK, so no dash needed unless it's modifying another noun, like "rate-limited requests"